_redirects

```diff
@@ -0,0 +1,3 @@
+/impressum.html /impressum.html 200
```

Comment:
Can you please link the Netlify documentation for redirects in the commit message?